app/src/main/java/com/quran/labs/androidquran/data/QuranDataProvider.java

```diff
@@ -182,6 +182,12 @@ private Cursor getSuggestions(String query) {
       }
     }
 
+    if (context != null) {
+      mc.addRow(new Object[] {
```

Comment:
maybe can improve this - only show for example if `queryIsArabic`, or if not all databases were searched, or if more than a certain number of results were found.